src/components/GoodsList.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export class GoodsList extends React.Component {
+  state = {
+    isReversed: false,
+    sortBy: 'default',
+  }
+
+  reverse = () => {
+    this.setState(state => ({ isReversed: !state.isReversed }));
+  }
+
+  sortByLetter = () => {
+    this.setState({ sortBy: 'letter' });
+    this.setState({ isReversed: false });
+  }
```

Comment:
```suggestion
  sortByLetter = () => {
    this.setState({
      sortBy: 'letter',
      isReversed: false
    });
  }
```
Why are you using two `setStates` here?